system-addon/lib/PlacesFeed.jsm

```diff
@@ -207,9 +215,23 @@ class PlacesFeed {
       case at.DELETE_HISTORY_URL:
         NewTabUtils.activityStreamLinks.deleteHistoryEntry(action.data);
         break;
+      case at.OPEN_NEW_WINDOW:
```

Comment:
I might be inclined to suggest creating a new `WindowFeed` or something like that... but because there is a lot of overhead to creating new jsms, maybe just leave this for now.